job-service-scheduled-executor/src/main/java/com/github/jobservice/scheduledexecutor/WorkerAction.java

```diff
@@ -121,6 +122,19 @@ public void setTargetPipe(final String targetPipe) {
         this.targetPipe = targetPipe;
     }
 
+    public WorkerAction correlationId(final String correlationId) {
+        this.correlationId = correlationId;
+        return this;
+    }
+
+    public String getCorrelationId() {
+        return correlationId;
+    }
+
+    public void setCorrelationId(final String correlationId) {
+        this.correlationId = correlationId;
+    }
+
     @Override
     public boolean equals(final Object o) {
```

Comment:
- The `equals()` method has not been updated for the addition of the correlation id.
- Same with the `hashCode()` method below.
- And `toString()` ...